pkg/scheduler/plugins/loadaware/load_aware.go

```diff
@@ -142,10 +141,9 @@ func (p *Plugin) Filter(ctx context.Context, state *framework.CycleState, pod *c
 		return framework.NewStatus(framework.Error, err.Error())
 	}
 
-	if p.args.FilterExpiredNodeMetrics != nil && *p.args.FilterExpiredNodeMetrics && p.args.NodeMetricExpirationSeconds != nil {
-		if isNodeMetricExpired(nodeMetric, *p.args.NodeMetricExpirationSeconds) {
-			return framework.NewStatus(framework.Unschedulable, ErrReasonNodeMetricExpired)
-		}
+	if p.args.FilterExpiredNodeMetrics != nil && *p.args.FilterExpiredNodeMetrics &&
```

Comment:
Maybe we should mark the args.FiterExpiredNodeMetrics always as true? WDYT? @lucming

Reply:
> Maybe we should mark the args.FiterExpiredNodeMetrics always as true? WDYT? @lucming

agree with you, or how about just removing args.FiterExpiredNodeMetrics?

Reply:
If it is removed, there will be some compatibility issues. For example, users who are already using it need to modify the configuration. Of course, you can also consider implementing a custom conversion to solve this compatibility problem. You can refer to:

https://github.com/koordinator-sh/koordinator/blob/11f1bbb4aee81838b7d4b1c46fe481df7f379a67/pkg/descheduler/apis/config/v1alpha2/conversion_plugins.go#L25-L47

Reply:
I've marked FiterExpiredNodeMetrics as deprecated, this field can be removed when the api is upgraded next time